processor/memorylimiterprocessor/config.go

```diff
@@ -6,11 +6,21 @@
 package memorylimiterprocessor // import "go.opentelemetry.io/collector/processor/memorylimiterprocessor"
 
 import (
+	"errors"
 	"time"
 
 	"go.opentelemetry.io/collector/component"
 )
 
+var (
+	errCheckIntervalOutOfRange        = errors.New("check_interval must be greater than zero")
+	errLimitOutOfRange                = errors.New("limit_mib or limit_percentage must be greater than zero")
+	errSpikeLimitOutOfRange           = errors.New("spike_limit_mib must be smaller than limit_mib")
+	errSpikeLimitPercentageOutOfRange = errors.New("spike_limit_percentage must be smaller than limit_percentage")
+	errLimitPercentageOutOfRange      = errors.New(
+		"limit_percentage and spike_limit_percentage must be greater than zero and less than or equal to hundred")
```

Comment:
use `"'check_interval' must be greater than zero"`

Reply:
done